csi.proto

```diff
@@ -1257,6 +1300,35 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  enum Condition {
+    // Volume health condition is unknown; treat it as healthy.
+    UNKNOWN = 0;
+    // Volume is healthy.
+    HEALTHY = 1;
+    // Volume is temporarily unhealthy, but will likely become healthy.
+    TEMPORARY_UNHEALTHY = 2;
+    // Volume will fail soon. Move your data off and replace volume or
+    // you may lose your data.
+    FATAL_IMMINENT = 3;
+    // Volume is unusable. Data loss is likely.
```

Comment:
Volume is not accessible currently, and there is reason to believe the outage may be permanent. Examples: unrecoverable hardware failure, natural disasters, damage to equipment.

Reply:
Done